diagnostics/networking_diagnostics.bat

```diff
@@ -0,0 +1,37 @@
+set networking_folder=".\networking_logs"
+set neworking_logs_zip=".\WslNetworkingLogs.zip"
+
+mkdir %networking_folder%
+
+wsl.exe tr -d "\r" ^| bash < ./networking.sh > %networking_folder%\wsl_network_configuration_before.log
+powershell Get-NetRoute > %networking_folder%\get_netroute.log
+
+powershell invoke-expression 'cmd /c start powershell -Command { .\collect-wsl-logs.ps1 }'
+powershell invoke-expression 'cmd /c start powershell -Command { "wsl.exe sudo tcpdump -n -i any > tcpdump.log" }'
+
+wpr -start .\wsl_networking.wprp -filemode -instanceName wpr_networking

```

Comment:
Instead of having two log collection running at the same time, let's pass this custom log profile to collect-wsl-logs instead

Reply:
Currently wsl_networking.wprp does not include the trace providers that wsl.wprp has - we need the traces providers from both wsl.wprp and wsl_networking.wprp

thoughts? should we merge wsl.wprp and wsl_networking.wprp in a single file?

Reply:
I think we should keep wsl.wprp as it currently to avoid creating too much noise in log files, but I think adding all the profiles from wsl.wprp into wsl_networking.wprp would make sense here

Reply:
fixed in iteration 3